stable/contour/templates/crd.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
Should the CRDs use the [crd-install hook](https://helm.sh/docs/developing_charts/#hooks)?